src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -198,6 +198,8 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
 	<AndroidGenerateLayoutBindings Condition=" '$(AndroidGenerateLayoutBindings)' == '' ">False</AndroidGenerateLayoutBindings>
 	<AndroidFragmentType Condition=" '$(AndroidFragmentType)' == '' ">Android.App.Fragment</AndroidFragmentType>
 	<AndroidEnableAssemblyCompression Condition=" '$(AndroidEnableAssemblyCompression)' == '' ">True</AndroidEnableAssemblyCompression>
+	<AndroidIncludeWrapSh Condition=" '$(AndroidIncludeWrapSh)' == '' ">False</AndroidIncludeWrapSh>
+	<AndroidCheckedBuild Condition=" '$(AndroidCheckedBuild)' == '' "></AndroidCheckedBuild>
```

Comment:
I think you can remove this line if it defaults to empty.

Reply:
@jonpryor thought it was good to put it in when I asked a few weeks back. @jonpryor, thoughts?